### ISSUE TITLE
[1.11-1.12.2]Crash/Conflict on Login

### ISSUE THREAD

**Issue by EvilDylan** [Nuchaz/BiblioCraft#486]
[crash-2017-07-22_12.12.29-server.txt](https://github.com/Nuchaz/BiblioCraft/files/1167476/crash-2017-07-22_12.12.29-server.txt)
(I hope that attachment works, new here.)

Upon trying to recreate an issue my friend had with bibliocraft shelves rotating on login and losing all of their items, I placed one such shelf at spawn in a singleplayer world. After logging back in, attempting to log back in twice resulted in a crash as seen above. The crash specifically mentions a tile entity shelf, so I believe it is related to this mod.
I am unsure if this is a conflict. I do not have optifine or any other rendering mods, but I do have a list of other mods, including bibliowoods[natura] and bibliowoods[bop], if it is relevant.

EDIT: Tested with 1.12.2 versions, issue still occurs; see post below.

**Comment by Nuchaz** [comment]
It is acting as if the shelf tile entity isn't properly registered. The only potential thing I can maybe think is the shelf is disabled in the config file, but maybe I don't have something right so its only disabling part of the shelf, like the tile entity. One option I can think of is to try deleting the config file and letting it regenerate and see if that helps.

**Comment by EvilDylan** [comment]
Sorry for the late response! I updated Bibliocraft and replaced the config.
However! I found the crash was actually caused in part by my own error-it appears that placing the Journal from the mod Astral Sorcery will cause this crash on login. Apologies again.

**Comment by EvilDylan** [comment]
@Nuchaz I have tested this issue more extensively and come to the conclusion that it is a conflict rather than an issue with your mod specifically. I have updated the title to reflect such.

Forge is version 1.11.2-forge1.11.2-13.20.1.2393
The only mods present in this environment are Bibliocraft and Real World (Exists [here](http://10paksmods.net/realworld.html).)

I mention it here because Bibliocraft is the mod mentioned in all crash logs related to the issue.
The only changes made to the Bibliocraft config were to add dragonbreath as an item to display on potion shelves. No changes were made to the Real World config.

The issue occurs with shelves. The issue does not occur with Bibliocraft alone, thus, it is likely a conflict.
The issue progresses as such: a spruce shelf is placed, and three of four spaces are filled with random items (I chose flint, rabbit hide, and coal.) The world is saved and exited, but the game itself is not closed. The world is reloaded.
Most often, the world will crash instantly upon reloading, resulting in the following log:
[crash-2017-08-14_15.18.43-server.txt](https://github.com/Nuchaz/BiblioCraft/files/1223234/crash-2017-08-14_15.18.43-server.txt)
I have noticed this most when the shelf is placed facing north (when the wall it is placed against is south,) however I am unsure if there is a correlation here.

**When the game does _not_ crash**, upon reloading the game the shelf will be rotated to face east (the wall it would be against to the west,) deleting any items that have been placed upon it.
![2017-08-14_15 29 39](https://user-images.githubusercontent.com/26489839/29288486-ee6fd63c-8106-11e7-8bb4-b0488eeaa331.png)
(Log out/log in)
![2017-08-14_15 29 54](https://user-images.githubusercontent.com/26489839/29288485-ee690910-8106-11e7-813d-7720c947df70.png)

I would speculate that it has something to do with Real World having its own shelves, but I am uncertain if this is the case.

I hope this report is thorough enough to draw a conclusion from. It might also be worth mentioning that Real World is a combination of the old Plant Mega Pack and Decorations Mega Pack, so if this issue has come up with one of these two before it might narrow down where the problem lies.
